examples/oci/vpn_sandbox/instances.tf

```diff
@@ -0,0 +1,83 @@
+resource "oci_core_instance" "server01" {
+  availability_domain = "${oci_core_subnet.sub01be.availability_domain}"
```

Comment:
can we look to use the `count` syntax to reduce duplication here? We can also target a similar thing for the subnets and refer to the correct instance via a convention, right?

Reply:
Thanks @kilokahn for the feedback! Yes, sure thing! I will rename it to ```libreswan_ipsec_vpn_basic```, as the idea is for it to be a very basic (starter kind of) example. I will add an advanced version with ```modules``` and ```count``` later on (with ```_advanced```).